warnings on missing `override`

When using ITK 5.2.0, CLang warns 
https://travis-ci.com/github/UCL/PETPVC/jobs/509265841#L3996-L3999
```
/Users/travis/build/UCL/PETPVC/lib/petpvcGTMImageFilter.h:84:18: warning: 'GenerateData' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
```
Not sure when `override` was introduced. in ITK but presumably we need a version check

--- a/lib/petpvcDiscreteIYPVCImageFilter.h
+++ b/lib/petpvcDiscreteIYPVCImageFilter.h
@@ -144,7 +144,7 @@ protected:
     ~DiscreteIYPVCImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matGTM;

--- a/lib/petpvcFuzzyCorrectionFilter.h
+++ b/lib/petpvcFuzzyCorrectionFilter.h
@@ -72,7 +72,7 @@ protected:
     ~FuzzyCorrectionFilter();
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
 
 private:

--- a/lib/petpvcGTMImageFilter.h
+++ b/lib/petpvcGTMImageFilter.h
@@ -81,7 +81,7 @@ protected:
     ~GTMImageFilter();
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
 private:
     GTMImageFilter(const Self &); //purposely not implemented

--- a/lib/petpvcIntraRegRLImageFilter.h
+++ b/lib/petpvcIntraRegRLImageFilter.h
@@ -128,7 +128,7 @@ protected:
     ~IntraRegRLImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     ITKVectorType m_vecVariance;
     unsigned int m_nIterations;

--- a/lib/petpvcIntraRegVCImageFilter.h
+++ b/lib/petpvcIntraRegVCImageFilter.h
@@ -139,7 +139,7 @@ protected:
     ~IntraRegVCImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     ITKVectorType m_vecVariance;
     unsigned int m_nIterations;

--- a/lib/petpvcIterativeYangPVCImageFilter.h
+++ b/lib/petpvcIterativeYangPVCImageFilter.h
@@ -136,7 +136,7 @@ protected:
     ~IterativeYangPVCImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matGTM;

--- a/lib/petpvcLabbeImageFilter.h
+++ b/lib/petpvcLabbeImageFilter.h
@@ -82,7 +82,7 @@ protected:
     ~LabbeImageFilter();
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
 private:
     LabbeImageFilter(const Self &); //purposely not implemented

--- a/lib/petpvcLabbeMTCPVCImageFilter.h
+++ b/lib/petpvcLabbeMTCPVCImageFilter.h
@@ -133,7 +133,7 @@ protected:
     ~LabbeMTCPVCImageFilter() {}
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matGTM;

--- a/lib/petpvcLabbePVCImageFilter.h
+++ b/lib/petpvcLabbePVCImageFilter.h
@@ -125,7 +125,7 @@ protected:
     ~LabbePVCImageFilter() {}
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matLabbe;

--- a/lib/petpvcLabbeRBVPVCImageFilter.h
+++ b/lib/petpvcLabbeRBVPVCImageFilter.h
@@ -137,7 +137,7 @@ protected:
     ~LabbeRBVPVCImageFilter() {}
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matLabbe;

--- a/lib/petpvcMTCPVCImageFilter.h
+++ b/lib/petpvcMTCPVCImageFilter.h
@@ -133,7 +133,7 @@ protected:
     ~MTCPVCImageFilter() {}
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matGTM;

--- a/lib/petpvcMullerGartnerImageFilter.h
+++ b/lib/petpvcMullerGartnerImageFilter.h
@@ -154,7 +154,7 @@ protected:
     }
 
     //void ThreadedGenerateData(const OutputImageRegionType& outputRegionForThread, int threadId );
-    void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
 protected:
 

--- a/lib/petpvcRBVPVCImageFilter.h
+++ b/lib/petpvcRBVPVCImageFilter.h
@@ -136,7 +136,7 @@ protected:
     ~RBVPVCImageFilter() {}
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matGTM;

--- a/lib/petpvcRLPVCImageFilter.h
+++ b/lib/petpvcRLPVCImageFilter.h
@@ -112,7 +112,7 @@ protected:
     ~RichardsonLucyPVCImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     ITKVectorType m_vecVariance;
     unsigned int m_nIterations;

--- a/lib/petpvcRegionConvolutionImageFilter.h
+++ b/lib/petpvcRegionConvolutionImageFilter.h
@@ -114,7 +114,7 @@ protected:
     ~RegionConvolutionPVCImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     ITKVectorType m_vecVariance;
     bool m_bVerbose;

--- a/lib/petpvcRoussetPVCImageFilter.h
+++ b/lib/petpvcRoussetPVCImageFilter.h
@@ -122,7 +122,7 @@ protected:
     ~RoussetPVCImageFilter() {}
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matGTM;

--- a/lib/petpvcSTCPVCImageFilter.h
+++ b/lib/petpvcSTCPVCImageFilter.h
@@ -162,7 +162,7 @@ protected:
     ~STCPVCImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     VectorType m_vecRegMeansPVCorr;
     MatrixType m_matGTM;

--- a/lib/petpvcVanCittertPVCImageFilter.h
+++ b/lib/petpvcVanCittertPVCImageFilter.h
@@ -119,7 +119,7 @@ protected:
     ~VanCittertPVCImageFilter() {};
 
     /** Does the real work. */
-    virtual void GenerateData();
+    virtual void GenerateData() ITK_OVERRIDE;
 
     ITKVectorType m_vecVariance;
     unsigned int m_nIterations;

